integration/simulation/transaction_injector.go

```diff
@@ -241,101 +239,12 @@ func (ti *TransactionInjector) issueRandomTransfers() {
 
 // issueRandomDeposits creates and issues a number of transactions proportional to the simulation time, such that they can be processed
 func (ti *TransactionInjector) issueRandomDeposits() {
-	var err error
```

Comment:
If there's no deposits will there be enough obx on the l2 to issue txs ?

Reply:
OBX shouldn't be impacted, but in fairness I should probably check in detail. The bridge is only dealing with HOC and POC, or at least it seems this way.

Reply:
Yep, these are only for HOC deposits, you're right. The sim injectors are likely using the prefunded keys